integreat_cms/core/formats/en/formats.py

```diff
@@ -0,0 +1,15 @@
+"""
+This file contains locale formats to override the default ones.
+See :setting:`django:FORMAT_MODULE_PATH` and :attr:`~integreat_cms.core.settings.FORMAT_MODULE_PATH` for more information.
+"""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import Final
+
+#: The default formatting to use for displaying datetime fields in any part of the system when using the English locale.
+DATETIME_FORMAT: Final[str] = "d.m.Y, H:i"
+SHORT_DATE_FORMAT: Final[str] = "d.m.Y"
```

Comment:
It was necessary to add a format for English default UI language to overwrite the long string format in media library